examples/CantileverDistributedLoad/cantilever_nonconservative_distributed_load.py

```diff
@@ -0,0 +1,250 @@
+from matplotlib import pyplot as plt
+import numpy as np
+import elastica as ea
+import json
+
+from cantilever_distrubuted_load_postprecessing import (
+    plot_video_with_surface,
+    find_tip_position,
+    adjust_square_cross_section,
+    NonconserativeForce,
+)
+
+
+def cantilever_subjected_to_a_nonconservative_load(
+    n_elem,
+    base_length,
+    side_length,
+    base_radius,
+    youngs_modulus,
+    dimentionless_varible,
+    Animation=False,
+    PLOT_FIGURE_Equilibrium=False,
```

Comment:
Again change names of function arguments and variables to `lower_case_with_underscores`.